addon/ng2/blueprints/ng2/files/__path__/test.ts

```diff
@@ -0,0 +1,41 @@
+
+/*global jasmine, __karma__, window*/
+require('core-js/es6');
+require('core-js/es7/reflect');
+
+// Typescript emit helpers polyfill
+require('ts-helpers');
```

Comment:
I'm reading this as then replace ts-helpers simply with tsconfig `importHelpers: true`?